src/at2.proto

```diff
@@ -4,19 +4,17 @@ package at2;
 service AT2 {
```

Comment:
I wonder whether at a given moment you'd want to add versions to your API. Or will you do that with the path? Like going to `api-url/v1`, aso?

And at some moment you'll have to define that it's a 'stable' version. My experience tells me that you should start as soon as possible to think about versioning, specifically for APIs. Of course you won't need it in the beginning, and you'll change/break it all the time (going from 0.1 to 0.2 to 0.3). But it puts you in the correct mindset to think how you can avoid future breakages, specifically once you reach a 1.0.

So here it should just state that it's a 0.1 version somewhere. Or 0.2, as you just broke the API ;)

Reply:
> I wonder whether at a given moment you'd want to add versions to your API. Or will you do that with the path? Like going to `api-url/v1`, aso?

Yep, I was thinking of doing it using the path and the crate version.

> So here it should just state that it's a 0.1 version somewhere. Or 0.2, as you just broke the API ;)

As it was never deployed publicly, I'm keeping it as `0.1` :)

Reply:
As an exercise, I'd consider merging with master as 'deploy'. At least in my experience, I never _really_ thought about versioning until I started harassing myself for it.